python/tskit/metadata.py

```diff
@@ -163,21 +163,8 @@ def binary_format_validator(validator, types, instance, schema):
         )
 
 
-def required_validator(validator, required, instance, schema):
```

Comment:
Merge artefact

Reply:
I've reverted the documentation filer, and both `metadata.py` and `test_metadata.py`. Should have been more careful about checking the patch. Thank you for taking a careful look into it.